app/controllers/detailedvaluationrequest/DvrController.scala

```diff
@@ -53,25 +54,35 @@ class DvrController @Inject()(
 
   def myOrganisationRequestDetailValuationCheck(
         propertyLinkSubmissionId: String,
-        valuationId: Long): Action[AnyContent] =
-    detailedValuationRequestCheck(propertyLinkSubmissionId, valuationId, true)
+        valuationId: Long,
+        uarn: Long): Action[AnyContent] =
+    detailedValuationRequestCheck(propertyLinkSubmissionId, valuationId, uarn, true)
 
-  def myClientsRequestDetailValuationCheck(propertyLinkSubmissionId: String, valuationId: Long): Action[AnyContent] =
-    detailedValuationRequestCheck(propertyLinkSubmissionId, valuationId, false)
+  def myClientsRequestDetailValuationCheck(
+        propertyLinkSubmissionId: String,
+        valuationId: Long,
+        uarn: Long): Action[AnyContent] =
+    detailedValuationRequestCheck(propertyLinkSubmissionId, valuationId, uarn, false)
 
   private def detailedValuationRequestCheck(
         propertyLinkSubmissionId: String,
         valuationId: Long,
+        uarn: Long,
         owner: Boolean): Action[AnyContent] = authenticated.async { implicit request =>
     val pLink =
       if (owner) propertyLinks.getOwnerAssessments(propertyLinkSubmissionId)
       else propertyLinks.getClientAssessments(propertyLinkSubmissionId)
 
     pLink.flatMap {
-      case Some(link) =>
+      case Some(link: ApiAssessments) =>
```

Comment:
Why Some(link: ApiAssessments) and not Some(link) ? 
Is it possible to be anything else?

Reply:
I should have removed that. I added that because I wanted to see what type "link" was. I will remove now

Reply:
gone